pkg/middleware/auth_test.go

```diff
@@ -353,3 +356,87 @@ func TestRemoveForceLoginparams(t *testing.T) {
 		})
 	}
 }
+
+func TestCanAdminPlugin(t *testing.T) {
+	type testCase struct {
+		desc       string
+		url        string
+		isSignedIn bool
+		orgRole    org.RoleType
+		expCode    int
+		expReached bool
+	}
+
+	ac := &actest.FakeAccessControl{}
```

Comment:
Is there a way we can use the actual implementation?
tried `acimpl.ProvideAccessControlTest()` but cyclic import is not allowed in test.

Reply:
you may try to change this file package name to `middleware_test`, not sure if you won't get a cyclic dependency that way but in any case, I think you are good with the fake implementation.